ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/ReadOnlySqlLedger.scala

```diff
@@ -34,9 +34,8 @@ object ReadOnlySqlLedger {
   val noOfStreamingConnections = 2
 
   //jdbcUrl must have the user/password encoded in form of: "jdbc:postgresql://localhost/test?user=fred&password=secret"
-  def apply(jdbcUrl: String, ledgerId: Option[LedgerId])(
-      implicit mat: Materializer,
-      mm: MetricsManager): Future[ReadOnlyLedger] = {
+  def apply(
+      jdbcUrl: String)(implicit mat: Materializer, mm: MetricsManager): Future[ReadOnlyLedger] = {
     implicit val ec: ExecutionContext = DEC
```

Comment:
Not part of this contribution but probably worth thinking of making this explicit. Having an implicit parasitic execution context seems like a gun pointed at our own foot.